### ISSUE TITLE
DEB (non-CLI) not starting due to missing dependency libadwaita

### ISSUE THREAD

**Issue by Centurix** [DevToys-app/DevToys#1206]
### Current behavior

Installing from x86 DEB package misses the libadwaita dependency. Post installation of the package resolves the issue.

Distro: Linux Mint 21.3

### How to reproduce it (as minimally and precisely as possible)

1. Download from [here](https://github.com/DevToys-app/DevToys/releases/download/v2.0.1.0/devtoys_linux_x64.deb)
2. Install
3. Run `devtoys`

### Expected behavior

Application should start

### Workaround

`sudo apt install libadwaita-1-0`

Application will start.

### Affected platforms

Linux

### Affected DevToys kind

DevToys (app with GUI)

### DevToys Version

v2.0-preview.1

### Relevant Assets/Logs

Unhandled exception. System.TypeInitializationException: The type initializer for 'Adw.Application' threw an exception.
 ---> System.DllNotFoundException: Unable to load shared library 'libadwaita-1.so.0' or one of its dependencies. In order to help diagnose loading problems, consider using a tool like strace. If you're using glibc, consider setting the LD_DEBUG environment variable: 
/opt/devtoys/devtoys/libadwaita-1.so.0: cannot open shared object file: No such file or directory
/opt/devtoys/devtoys/liblibadwaita-1.so.0: cannot open shared object file: No such file or directory
/opt/devtoys/devtoys/libadwaita-1.so.0.so: cannot open shared object file: No such file or directory
/opt/devtoys/devtoys/liblibadwaita-1.so.0.so: cannot open shared object file: No such file or directory

   at System.Runtime.InteropServices.NativeLibrary.LoadLibraryByName(String libraryName, Assembly assembly, Nullable`1 searchPath, Boolean throwOnError)
   at Adw.Internal.ImportResolver.Resolve(String libraryName, Assembly assembly, Nullable`1 searchPath)
   at System.Runtime.InteropServices.NativeLibrary.LoadLibraryCallbackStub(String libraryName, Assembly assembly, Boolean hasDllImportSearchPathFlags, UInt32 dllImportSearchPathFlags)
   at Adw.Internal.Functions.Init()
   at Adw.Module.Initialize()
   at Adw.Application..cctor()
   --- End of inner exception stack trace ---
   at Adw.Application.New(String applicationId, ApplicationFlags flags)
   at DevToys.Linux.LinuxProgram..ctor() in /home/etienne/RiderProjects/Publish/submodules/DevToys/src/app/dev/platforms/desktop/DevToys.Linux/LinuxProgram.cs:line 34
   at DevToys.Linux.Program.Main(String[] args) in /home/etienne/RiderProjects/Publish/submodules/DevToys/src/app/dev/platforms/desktop/DevToys.Linux/Program.cs:line 9
[1]    11204 IOT instruction (core dumped)  devtoys

**Comment by Spitfire1900** [comment]
Reproduced with the Portable zip download for x64 as well.

```
|@ ./DevToys.Linux
Unhandled exception. System.TypeInitializationException: The type initializer for 'Adw.Application' threw an exception.
 ---> System.DllNotFoundException: Unable to load shared library 'libadwaita-1.so.0' or one of its dependencies. In order to help diagnose loading problems, consider using a tool like strace. If you're using glibc, consider setting the LD_DEBUG environment variable: 
/home/kyle/Downloads/devtoys/devtoys_linux_x64_portable/libadwaita-1.so.0: cannot open shared object file: No such file or directory
/home/kyle/Downloads/devtoys/devtoys_linux_x64_portable/liblibadwaita-1.so.0: cannot open shared object file: No such file or directory
/home/kyle/Downloads/devtoys/devtoys_linux_x64_portable/libadwaita-1.so.0.so: cannot open shared object file: No such file or directory
/home/kyle/Downloads/devtoys/devtoys_linux_x64_portable/liblibadwaita-1.so.0.so: cannot open shared object file: No such file or directory

   at System.Runtime.InteropServices.NativeLibrary.LoadLibraryByName(String libraryName, Assembly assembly, Nullable`1 searchPath, Boolean throwOnError)
   at Adw.Internal.ImportResolver.Resolve(String libraryName, Assembly assembly, Nullable`1 searchPath)
   at System.Runtime.InteropServices.NativeLibrary.LoadLibraryCallbackStub(String libraryName, Assembly assembly, Boolean hasDllImportSearchPathFlags, UInt32 dllImportSearchPathFlags)
   at Adw.Internal.Functions.Init()
   at Adw.Module.Initialize()
   at Adw.Application..cctor()
   --- End of inner exception stack trace ---
   at Adw.Application.New(String applicationId, ApplicationFlags flags)
   at DevToys.Linux.LinuxProgram..ctor() in /home/etienne/RiderProjects/Publish/submodules/DevToys/src/app/dev/platforms/desktop/DevToys.Linux/LinuxProgram.cs:line 34
   at DevToys.Linux.Program.Main(String[] args) in /home/etienne/RiderProjects/Publish/submodules/DevToys/src/app/dev/platforms/desktop/DevToys.Linux/Program.cs:line 9

Aborted (core dumped)
```

**Comment by veler** [comment]
Hi,
Thanks for reporting this issue. We're adding `libadwaita-1-0` to the list of dependencies in CI (private).

**Comment by Spitfire1900** [comment]
@veler - is there a good way to tell when the fix for this issue has been published in a new release artifact?

**Comment by veler** [comment]
> @veler - is there a good way to tell when the fix for this issue has been published in a new release artifact?

It is available in v2.0.2.0 since yesterday: https://github.com/DevToys-app/DevToys/releases/tag/v2.0.2.0

It doesn't appear in the "details" of the release because the fix has been done on a private repository (our releasing pipeline), but I mentioned it briefly at the top.
![image](https://github.com/DevToys-app/DevToys/assets/3747805/ff1925ce-5b5c-43cd-8e95-f0d4c3b711fb)
